src/view/treeNodes/pullRequestNode.ts

```diff
@@ -64,7 +65,7 @@ export class PRNode extends TreeNode implements vscode.CommentingRangeProvider {
 		try {
 			const descriptionNode = new DescriptionNode(
 				this,
-				vscode.l10n.t('Description'),
+				`${vscode.l10n.t('Description')} #${this.pullRequestModel.number}`,
```

Comment:
I'm not a fan of having the PR number in the description node. When `githubPullRequests.showPullRequestNumberInTree` is false, it feels noisy and when `githubPullRequests.showPullRequestNumberInTree` is true it is redundant. I've pushed a change to revert this.

Reply:
Fair enough.